Standard Docker build environment broken?

### Setup
_Package Name:_ `Tvheadend` (for example)
_Package Version:_ 4.2.8-fixes (see PR #3623)

_NAS Model:_ Various
_NAS Architecture:_ Various
_DSM version:_ 6.1

### Expected behavior
Should compile in standard Docker build environment
[Edit: Formerly reported problems with VirtualBox builds had different rationale. Removed as obsolete.]

### Actual behavior
Builds on Standard Docker image (pulled via travis) fail with:
```
CMake Error at CMakeLists.txt:4 (cmake_minimum_required):
  CMake 3.1 or higher is required.  You are running version 3.0.2
``` 
See https://travis-ci.org/m4tt075/spksrc/builds/492410154 for more examples

### Steps to reproduce
Try building `tvh-4.2.8-fixes` with standard Docker image (pulled in via travis)

--- a/cross/gmp/Makefile
+++ b/cross/gmp/Makefile
@@ -11,8 +11,6 @@ HOMEPAGE = http://gmplib.org/
 COMMENT  = GMP is a free library for arbitrary precision arithmetic, operating on signed integers, rational numbers, and floating point numbers
 LICENSE  = LGPL
 
-include ../../mk/spksrc.cross-cc.mk
-
 GNU_CONFIGURE = 1
 ifeq ($(findstring $(ARCH), $(ARM5_ARCHES)),$(ARCH))
 CONFIGURE_ARGS = --target=arm-unknown-linux-gnueabi --host=arm-unknown-linux-gnueabi --disable-assembly
@@ -32,3 +30,5 @@ endif
 ifeq ($(findstring $(ARCH), $(x64_ARCHES)),$(ARCH))
 CONFIGURE_ARGS = --target=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu
 endif
+
+include ../../mk/spksrc.cross-cc.mk

